src/components/Home/HomeHeaderBackground.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Box, Container } from '@mui/material';
+
+import SearchForm from './SearchForm';
+import HomeBackground from '../../assets/landscapeImage.jpg';
```

Comment:
Instead of relative path, you can use this path `'@/assets/images/landscapeImage.png'`